lib/manager/terraform/readme.md

```diff
@@ -1,21 +1,38 @@
-Currently Terraform support is limited to Terraform registry sources and GitHub sources that include SemVer refs, e.g. like `github.com/hashicorp/example?ref=v1.0.0`.
+Currently, Terraform supports renovating the following dependencies, where sub points represent hosting options of the dependencies:
 
-Fixed versions like the following will receive a PR whenever there is a newer version available:
+- modules
+  - GitTags
+  - GithubTags
+  - TerraformRegistry ( Public and Private )
+- providers ( deprecated in Terraform 0.13.0 )
+  - TerraformRegistry ( Public and Private )
+- required_providers block ( Terraform >= 0.13.0)
+  - TerraformRegistry ( Public and Private )
+- required_version
+- helm_release
+  - chart repository ( Public and Private )
+- docker\_\*
+  - Docker registry ( Public and Private )
```

Comment:
Would a table be clearer than the current list?

Reply:
Let's leave as-is. It's a tree concept so not a perfect fit for table either